src/Modal/ModalPortal.tsx

```diff
@@ -15,18 +28,26 @@ export const Props = {
   getChildContext: t.maybe(t.Function)
 };
 
+type ContextWrapperProps = {
+  modal: () => JSX.Element,
+  getChildContext?: () => object
+}
+
 @props(Props)
-export default class ModalPortal extends React.Component {
+export default class ModalPortal extends React.Component<ModalPortalProps> {
+
+  private isOpen: boolean;
 
   render() {
     return null;
   }
 
   _ContextWrapper = (() => {
     const childContextTypes = this.props.childContextTypes || {};
-    return class ContextWrapper extends React.Component { // eslint-disable-line react/no-multi-comp
+    const getChildContext = this.props.getChildContext || (() => ({}));
+    return class ContextWrapper extends React.Component<ContextWrapperProps> { // eslint-disable-line react/no-multi-comp
       static childContextTypes = childContextTypes;
-      getChildContext = this.props.getChildContext || (() => ({}));
```

Comment:
don't know why it was working before without this change, but now when I run tests I get `this.props.getChildContext` equal to `undefined` 😕

Reply:
looks better as it is now anyway, we were probably relying on ambiguous babel behavior?